components/brave_wallet/browser/json_rpc_responses.idl

```diff
@@ -69,4 +69,11 @@ namespace json_rpc_responses {
     DOMString totalBalanceUsd;
     AnkrAssetBalance[] assets;
   };
+
+  dictionary PolkadotSystemChainResponse {
+    DOMString jsonrpc;
+    long id;
+    DOMString? result;
+    Error? error;
+  };
```

Comment:
Should be in a separate polkadot_rpc similar to
https://github.com/brave/brave-core/blob/master/components/brave_wallet/browser/cardano/cardano_rpc_blockfrost_api.idl#L6